<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Opp1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Opp1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="player">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Opp2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Opp2</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TM">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TM</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card1">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card2">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card3">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card5">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card6">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card7">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>251</y>
      <width>44</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card8">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card9">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card10">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card11">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card12">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>251</y>
      <width>45</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card13">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>251</y>
      <width>44</width>
      <height>69</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bidBox">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>504</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="turnButton">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>500</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bidButton">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>500</y>
      <width>62</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Bid</string>
    </property>
   </widget>
   <widget class="QWidget" name="table" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="play12">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play5">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play6">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play11">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play3">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play10">
    <property name="geometry">
     <rect>
      <x>493</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play9">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play4">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play2">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play13">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play8">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>330</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play7">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>330</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>table</zorder>
   <zorder>Opp1</zorder>
   <zorder>player</zorder>
   <zorder>Opp2</zorder>
   <zorder>TM</zorder>
   <zorder>card1</zorder>
   <zorder>card2</zorder>
   <zorder>card3</zorder>
   <zorder>card4</zorder>
   <zorder>card5</zorder>
   <zorder>card6</zorder>
   <zorder>card7</zorder>
   <zorder>card8</zorder>
   <zorder>card9</zorder>
   <zorder>card10</zorder>
   <zorder>card11</zorder>
   <zorder>card12</zorder>
   <zorder>card13</zorder>
   <zorder>textBox</zorder>
   <zorder>scoreLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>bidBox</zorder>
   <zorder>turnButton</zorder>
   <zorder>bidButton</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
